app/views/test/components/mentoring/mentor_solutions_lists/show.html.haml

```diff
@@ -4,4 +4,4 @@
   = select_tag :state, options_for_select(%w[Success Error Loading], params[:state])
   = submit_tag "Submit", name: nil
 %hr/
-= mentor_solutions_list({ endpoint: test_components_mentoring_mentor_solutions_list_solutions_path, query: { state: params[:state] }, retry: { retry: params[:state] == "Loading" } }, @sort_options)
+= mentor_solutions_list({ endpoint: test_components_mentoring_mentor_solutions_list_solutions_path, query: { state: params[:state] }, options: { retry: params[:state] == "Loading" } }, @sort_options)
```

Comment:
When https://github.com/exercism/v3-website/pull/82 is merged, this needs to be updated.